packages/web5/lib/src/dids/did_dht/did_dht.dart

```diff
@@ -14,6 +18,124 @@ class DidDht {
 
   static final resolver = DidMethodResolver(name: methodName, resolve: resolve);
 
+  static Future<BearerDid> create({
+    required AlgorithmId algorithm,
+    required KeyManager keyManager,
+    List<String>? alsoKnownAs,
+    List<String>? controllers,
+    String? gatewayUri,
+    bool? publish,
+    List<DidService>? services,
+    List<DidDhtRegisteredDidType>? types,
+    List<DidCreateVerificationMethod>? verificationMethods,
+  }) async {
+    // Generate random key material for the Identity Key.
+    final Jwk identityKeyUri = await Crypto.generatePrivateKey(algorithm);
+    final Jwk identityKey = await Crypto.computePublicKey(identityKeyUri);
```

Comment:
similar to [did_jwk](https://github.com/TBD54566975/web5-dart/blob/main/packages/web5/lib/src/dids/did_jwk/did_jwk.dart#L29) `keyManager` can be optional and we'll want to use it to generate the private key vs. `Crypto.generatePrivateKey` so that the key is stored in whatever concrete implementation of key manager is provided 

also the algorithm for identity key must be `Ed25519`. More info [here](https://did-dht.com/#identity-key)!

Reply:
Awesome possum, I'll make those changes shortly!

Reply:
Update has been pushed!